Cookbook/Technical-Documents/SupportEngineerRole.md

```diff
@@ -9,7 +9,10 @@ When an engineer is on support, these duties have higher priority over Squad dut
 
 * **Update #ios and #ios-usa-support Slack channels topics** The topics should be updated with the iOS Engineers in support for the current day. This update should be done first thing in the morning.
 * **Be available to answer any requests posted in the #ios and #ios-usa-support.** The engineer can either solve the request or delegate to someone who might know how to solve it.
-* **Pick up tickets from the iOS Platform backlog.** These are tickets that don't have any Squad as an owner and still need to be fixed. [Kanban board](https://babylonpartners.atlassian.net/secure/RapidBoard.jspa?rapidView=1100&projectKey=IOSP)
+* **Pick up tickets from the iOS Platform backlog.** These are tickets that don't have any Squad as an owner and still need to be fixed, or ones for which the Platform team might need some help with.
+   * Use this [Kanban board](https://babylonpartners.atlassian.net/secure/RapidBoard.jspa?rapidView=1100&projectKey=IOSP) to see the list of pending BAU tickets
+   * But first, ping the platform team directly on Slack to check with them if there is any urgent/blocker ticket that could use some help and would take priority
+   * At the end of your shift, be sure to post your progress on the ticket, for the next engineeer to pick up, and inform them of any outstanding work.
```

Comment:
Where should we post support updates? In `#ios`?

Reply:
Mmmmh might need better phrasing as I see the wording ambiguity now…
The sentence was meant to say "post it as a comment on the ticket in JIRA"

Reply:
I've seen updates on Slack, my bad 🙈

Reply:
Well _additionally_ posting it on Slack can actually be a good idea 👍  (always better to over-communicate than under-communicate #transparency 😉). But that should not prevent the update to be posted on the ticket in any case, which should remain the reference point for gathering any info about the task to be done 🙂

Reply:
```suggestion
   * At the end of your shift, be sure to post your progress on the task as a comment on the JIRA ticket, for the next engineer to pick up, and inform them of any outstanding work. Optionally, feel free to also post updates in Slack too as a FYI to the whole team
```
@sdaniel55 clearer?

Reply:
Yep, it's clearer this way :) Thanks 💖

Reply:
Not sure why I can't click on the "Add suggestion to batch" on my previous comment (while I can for other suggestions on this page), so trying again here

```suggestion
   * At the end of your shift, be sure to post your progress on the task by adding a comment on the JIRA ticket, for the next engineer to pick up, and inform them of any outstanding work. Optionally, feel free to also post updates in Slack too as a FYI to the whole team
```